bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiClientThingHandler.java

```diff
@@ -54,35 +63,31 @@
 @NonNullByDefault
 public class UniFiClientThingHandler extends UniFiBaseThingHandler<UniFiClient, UniFiClientThingConfig> {
 
-    public static boolean supportsThingType(ThingTypeUID thingTypeUID) {
-        return UniFiBindingConstants.THING_TYPE_WIRELESS_CLIENT.equals(thingTypeUID);
-    }
-
     private final Logger logger = LoggerFactory.getLogger(UniFiClientThingHandler.class);
 
     private UniFiClientThingConfig config = new UniFiClientThingConfig();
 
-    public UniFiClientThingHandler(Thing thing) {
+    public UniFiClientThingHandler(final Thing thing) {
         super(thing);
     }
 
     @Override
-    protected synchronized void initialize(UniFiClientThingConfig config) {
+    protected boolean initialize(final UniFiClientThingConfig config) {
         // mgb: called when the config changes
         logger.debug("Initializing the UniFi Client Handler with config = {}", config);
         if (!config.isValid()) {
             updateStatus(OFFLINE, CONFIGURATION_ERROR,
                     "You must define a MAC address, IP address, hostname or alias for this thing.");
```

Comment:
If you would like to add i18n texts for offline reasons etc. this is also a candidate.